spec/models/article_spec.rb

```diff
@@ -587,8 +588,9 @@ def build_and_validate_article(*args)
         time_str = scheduled_time.strftime("%d/%m/%Y %H:%M %z")
         scheduled_body_markdown = "---\ntitle: Title\npublished: true\npublished_at: #{time_str}\n---\n\n"
         frontmatter_scheduled_article = create(:article, body_markdown: scheduled_body_markdown)
-        new_body_markdown = "---\ntitle: Title\npublished: false\npublished_at:2022-12-05 18:00 +0300---\n\n"
+        new_body_markdown = "---\ntitle: Title\npublished: false\n---\n\n"
```

Comment:
Here we were setting published_at to the date that is still in the future, which is allowed, so the test was failing.
Removed `published_at` to make a test more meaningful.